index.html

```diff
@@ -90,199 +90,213 @@ <h1>Am Alphy glad to see you!</h1>
                 </ul>
             </div>
         </section>
-        
+
 
 
```

Comment:
- [ ] I kindly suggest that you remove the large margin of whitespace you have to the right of your web page. Your portfolio should be centered as seen from the Figma Design.

https://www.loom.com/share/300c79e162404e109a686eb6d3fd7655